vite.config.js

```diff
@@ -1,14 +1,22 @@
-import { sveltekit } from '@sveltejs/kit/vite'
-import { defineConfig } from 'vitest/config'
-import wasm from 'vite-plugin-wasm'
-import topLevelAwait from 'vite-plugin-top-level-await'
+import { sveltekit } from '@sveltejs/kit/vite';
+import { defineConfig } from 'vitest/config';
+import wasm from 'vite-plugin-wasm';
+import topLevelAwait from 'vite-plugin-top-level-await';
 
 export default defineConfig({
-	plugins: [sveltekit(), wasm(), topLevelAwait()],
+	plugins: [
+		sveltekit(),
+		wasm(),
+		topLevelAwait()
+	],
 	test: {
-		include: ['src/**/*.{test,spec}.{js,ts}']
+		include: ['src/**/*.{test,spec}.{js,ts}'],
 	},
-	ssr: {
-		noExternal: ['three']
-	}
-})
+	server: {
+		fs: {
```

Comment:
vite config is black magic that I don't understand. If this works then LGTM!

Reply:
yep it works. had to do it because the hmr with `pnpm dev` didn't work after restructuring the project to a pnpm workspace

vite is black magic and I normally hate magic in code... but it's so well done! lot of great tools in the javascript world but this one is definitely near the top of the list for me :)